### ISSUE TITLE
invalid ember-usable dependency

### ISSUE THREAD

**Issue by bartocc** [nickschot/ember-mobile-menu#97]
https://github.com/pzuraq/ember-usable#0d03a50 does not seem to exist anymore.

Is this package needed anyway ?

**Comment by nickschot** [comment]
Yes it is needed for newer versions, it also still does seem to exist: https://github.com/pzuraq/ember-usable/commit/0d03a500a2f49041a4ddff0bb05b077c3907ed7d . It is less than ideal that we're using a non-published package though.

**Comment by nickschot** [comment]
Ah! Ninja-edit, I have no experience with yarn 2. Would be easy enough to update it to use the full hash.
